packages/office-addin-sso/tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+    "compilerOptions": {
+        "target": "es6",
```

Comment:
If the tsconfig **complilerOptions.target** is "es6", will the add-in work on IE 11?

Reply:
I assume this should be OK, as this is the same setting we have in tsconfig for all other packages in Office-Addin-Scripts